CONTRIBUTING.md

```diff
@@ -126,16 +135,19 @@ When ready, if you have not already done so, sign a
 the PR.
 
 See [Reviewing and Merging Pull Requests for Istio](REVIEWING.md) for the PR review and
-merge process that is used by the maintainers of the project.
+merge process that we use.
 
 ## Issues
 
 GitHub issues can be used to report bugs or submit feature requests.
 
 When reporting a bug please include the following key pieces of information:
```

Comment:
this should probably point to or mirror
https://github.com/istio/issues/issues/new

Reply:
Done.